ecs-proxies-deploy-sandbox.yml

```diff
@@ -3,9 +3,15 @@
 #docker_service_memory: 512
 
 docker_service:
-
-  - name: sds-sandbox
+  - name: sds-api
```

Comment:
something seems wrong here, you've removed sandbox and deployed api with mock. Surely we need both sandbox and api deployed, but with different values for SDS_MOCK_LDAP_RESPONSE and SDS_MOCK_LDAP_MODE?